### PR TITLE
CLNTINT-450 Capture requests and cookies

### PR DESCRIPTION
**Pull request by HeatherStenson**
- Updates `Capture Requests` to `Capture requests and cookies`

### CODE REVIEW

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ warning: false
 
 ---
 
-If you are using APIs to build client-side applications like mobile apps, websites, or desktop applications, you may want to see the actual HTTP and HTTPS request traffic that's being sent and received in the application. Sometimes you might discover APIs that aren't even documented. You can capture this network traffic using the proxy that's built into the Postman app.
+If you are using APIs to build client-side applications like mobile apps, websites, or desktop applications, you may want to see the actual HTTP and HTTPS request traffic that's being sent and received in the application. Sometimes you might discover APIs that aren't even documented. You can capture network traffic, request responses, and cookies using the proxy that's built into the Postman app.
```
Comment (shashankawasthi88) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:35]: request, response and cookies are all part of network traffic. Would we want to rephrase this?

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 2]
```diff
@@ -83,9 +83,9 @@ The proxy is now running, but it will not capture traffic until you start a prox
 
 After the proxy is running, you can start a proxy debug session. This is a time-bound session where captured traffic is saved in an entry in the **History** tab, and optionally into a collection. Once a proxy debug session is started, you can then pause, restart, or stop it. Before you start the session, you can select what traffic you want to go into it.
 
-> You can only have a single proxy or interceptor debug session running at the same time.
+> You can only have a single proxy or Interceptor debug session running at the same time.
```
Comment (shashankawasthi88) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:86]: Why is proxy with a 'p' vs Interceptor with 'I' ?

**Review by shashankawasthi88 — COMMENTED** [review]
On the interceptor page in the Sync cookie section, we should add the new Sync cookie snapshot and explain the flow as well.

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md (hunk 2 above)
Comment (jkonrath-postman) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:86]: Interceptor is a branded term (there's an internal list that's shared between UX content, us, and marketing). I'm not sure if we should make proxy a branded term too - let's talk about it offline later.

File: src/pages/docs/getting-started/introduction.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ contextual_links:
 
 Welcome to the Postman docs! This is the place to find official information on how to use Postman in your API projects.
 
-<img alt="Postman full view" src="https://assets.postman.com/postman-docs/intro-postman-request-selected-v9.1.jpg"/>
+![![Postman full view](https://assets.postman.com/postman-docs/intro-postman-request-selected-v9.8.jpg)](https://assets.postman.com/postman-docs/intro-postman-request-selected-v9.8.jpg)
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/introduction.md:42]: I think this is the one place where we need a full-width screenshot. Can you do this showing the left sidebar with collections visible?

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md (hunk 1 above)
Comment (jkonrath-postman) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:35]: They're the same, but I think we should leave this for search SEO, so this page comes up if you search on cookies.

File: src/pages/docs/sending-requests/capturing-request-data/interceptor.md [hunk 4]
```diff
@@ -45,21 +45,21 @@ Once Interceptor is running in Chrome, you can start a debug session, which is a
     * [Adding a custom encryption key in the UI](#adding-a-custom-encryption-key-in-the-ui)
     * [Adding a custom encryption key with the pm API](#adding-a-custom-encryption-key-with-the-pm-api)
 * [Updating Interceptor](#updating-interceptor)
-* [Using Interceptor with Postman's Chrome app](#using-the-interceptor-with-postmans-chrome-app)
+* [Using Interceptor with Postman's Chrome app (deprecated)](#using-the-interceptor-with-postmans-chrome-app-deprecated)
```
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/capturing-request-data/interceptor.md:48]: Change that back (and the related heading) - I think we might have a link going to that heading, based on the traffic that gets in analytics.

File: src/pages/docs/sending-requests/capturing-request-data/interceptor.md [hunk 5]
```diff
@@ -45,21 +45,21 @@ Once Interceptor is running in Chrome, you can start a debug session, which is a
...
 
 To get started, install Postman Interceptor:
 
 1. [Download](https://go.pstmn.io/interceptor-download) Interceptor in the Chrome Web Store. If you already have the extension, ensure it is version v0.2.26 or later.
 
-1. In the Postman app, select the **Capture Requests** icon in the Postman footer.
+1. In the Postman app, select the **Capture requests and cookies** icon in the Postman footer.
 
-    <img src="https://assets.postman.com/postman-docs/proxy-capture-requests-button.jpg" alt="Capture requests button" width="300px"/>
+    <img src="https://assets.postman.com/postman-docs/proxy-capture-requests-button-v9.8.jpg" alt="Capture requests button" width="450px"/>
```
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/capturing-request-data/interceptor.md:58]: This and the next image look a bit fuzzy. This one is 334px wide, but the width is set at 450px, which might be the culprit. But the original was 500-some wide, and scaled down to 300, which might be it.

Also, can you show the cursor in the image to show what they should be selecting?

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 6]
```diff
@@ -68,12 +68,12 @@ In the steps below, you will use the Postman app's proxy feature to inspect HTTP
...
 To begin, start the proxy inside Postman:
 
-1. Select the **Capture Requests** icon in the Postman footer.
+1. Select the **Capture requests and cookies** icon in the Postman footer.
 
-    <img src="https://assets.postman.com/postman-docs/proxy-capture-requests-button.jpg" alt="Capture requests button" width="300px"/>
-1. On the **Capture Requests** window, select the **Via Proxy** tab.
-    <img src="https://assets.postman.com/postman-docs/capture-via-proxy-tab.jpg" alt="Capture via proxy tab" width="300px"/>
-1. In the upper right, select **Enable Proxy**.
+    <img src="https://assets.postman.com/postman-docs/proxy-capture-requests-button-v9.8.jpg" alt="Capture requests button" width="450px"/>
```
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:73]: See my comments in Interceptor - these look a bit fuzzy.

File: src/pages/docs/sending-requests/capturing-request-data/interceptor.md [hunk 7]
```diff
@@ -73,7 +73,7 @@ After Interceptor is connected, you can then start an Interceptor debug session,
 
 > You can only have a single proxy or interceptor debug session running at the same time.
 
-1. Go to the **Via Incerceptor** tab of the **Capture Requests** window.
+1. Go to the **Via Incerceptor** tab of the **Capture requests and cookies** window.
```
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/capturing-request-data/interceptor.md:76]:
```suggestion
1. Go to the **Via Interceptor** tab of the **Capture requests and cookies** window.
```

**Review by jkonrath-postman — CHANGES_REQUESTED** [review]
A few changes on here.
 @shashankawasthi88  - on the comment about the added flow, if that can't be done quick, we'll open a second ticket on this, because I want to get these changes in ASAP.

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md (hunk 6 above)
Comment (mark-anthony-postman) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:73]: Did you take the images in retina mode (on your macbook screen)? That might help. Part of the problem is that there are small graphics being magnified.

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 8]
```diff
@@ -68,12 +68,12 @@ In the steps below, you will use the Postman app's proxy feature to inspect HTTP
...
+1. In the **Capture requests and cookies** window, select the **Via Proxy** tab.
+    <img src="https://assets.postman.com/postman-docs/capture-via-proxy-tab-v9.8.jpg" alt="Capture via proxy tab" width="300px"/>
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:75]: Add a blank  line above and below images to ensure correct spacing between numbered steps.

**Review by mark-anthony-postman — APPROVED** [review]
Let me know if you need help on screenshots 📸  
Otherwise LGTM 🌈

File: src/pages/docs/sending-requests/capturing-request-data/interceptor.md (hunk 7 above)
Comment (HeatherStenson) [reply src/pages/docs/sending-requests/capturing-request-data/interceptor.md:76]: Omg 🙈

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md (hunk 6 above)
Comment (HeatherStenson) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:73]: > Did you take the images in retina mode (on your macbook screen)

Oh! I'm not so that might help. Also realized that I'm stretching the images out larger than it's original size (per @jkonrath-postman's [comment](https://github.com/postmanlabs/postman-docs/pull/3828#discussion_r787879673)), which no doubt adds to the problem. I'll get these fixed!

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md (hunk 2 above)
Comment (HeatherStenson) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:86]: Per team conversation, I'll revert this to `interceptor` since it's not specifically referring to a named feature in Postman in this case.

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md (hunk 1 above)
Comment (HeatherStenson) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:35]: I'll rephrase this to be:

> You can capture network traffic, including requests, responses, and cookies, using the proxy that's built into the Postman app.

**Comment by HeatherStenson** [comment]
> On the interceptor page in the Sync cookie section, we should add the new Sync cookie snapshot and explain the flow as well.

@shashankawasthi88 After looking over the changes needed to add the new info about syncing cookies to the Interceptor page, I think that's sufficient work to warrant a separate issue as suggested by @jkonrath-postman. I will create a new ticket and tag you in it, and will ask you to review the new PR once it's ready.

**Review by claireriley — APPROVED** [review]
LGTM!
